go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/Telmate/proxmox-api-go
+module github.com/KnKay/proxmox-api-go
```

Comment:
I don't think you want to make this change.

Reply:
You are right. Was for double checking a module relying on this. 
Thought the MR was rejected after the first coment (not the most experienced github user)